File: C++/p117.cpp
#include <cstdio>
#include <iostream>
#include <string>

using namespace std;

int main(){
    int cantSaludos;
    string saludo;

    cin >> cantSaludos;

    for (int i = 0; i < cantSaludos; i++) {
        cin >> saludo  >> saludo;
        cout << "Hola, " << saludo << "." << endl;
    }

    return 0;
}

File: Java/P117.java
import java.util.*;

public class P117 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int cantidad;
        String saludo;

        cantidad = sc.nextInt();

        for (int i = 0; i < cantidad; i++) {
            sc.next();
            saludo = sc.next();
            System.out.println("Hola, " + saludo + ".");
        }
    }
}

